src/app-view.js

```diff
@@ -30,6 +31,10 @@ class AppView extends React.Component {
         this.checkCurrentSection();
         this.checkDisplayedComponent();
 
+        if (this.checkMobileDevice()) {
```

Comment:
another moment - we know that result of checkMobileDevice will not change during session, so shall we run one during init stage only and save in this.isMobileDevice property?

Reply:
Ideally, we should check on each window resize. While developing components in Styleguide developers can often toggle Chrome responsive mode. Styleguide UI should seamlessly adapt.